tests/cases/misc.rs

```diff
@@ -43,7 +43,9 @@ fn test_peer() {
         .unwrap();
     server.start();
     let port = server.bind_addrs()[0].1;
-    let ch = ChannelBuilder::new(env).connect(&format!("127.0.0.1:{}", port));
+    let cb = ChannelBuilder::new(env);
+    let cb = cb.load_balancing_policy(LbPolicy::RoundRobin);
```

Comment:
do we need to test different load balancer here?

Reply:
Removed this.